lib/apipie/swagger_generator.rb

```diff
@@ -63,36 +60,33 @@ def init_swagger_vars(version, lang, clear_warnings=false)
       # }
 
 
-      @swagger = {
-          swagger: '2.0',
-          info: {
-              title: "#{Apipie.configuration.app_name}",
-              description: "#{Apipie.app_info(version, lang)}#{Apipie.configuration.copyright}",
-              version: "#{version}",
-              "x-copyright" => Apipie.configuration.copyright,
-          },
-          basePath: Apipie.api_base_url(version),
-          consumes: [],
-          paths: {},
-          definitions: {},
-          tags: [],
+      @openapi_schema = {
+        openapi: '3.0.3',
+        info: {
+          title: "#{Apipie.configuration.app_name} (params in:#{json_body? ? 'body' : 'formData'})",
+          description: "#{Apipie.app_info(version, lang)}#{Apipie.configuration.copyright}",
+          version: version.to_s,
+          'x-copyright': Apipie.configuration.copyright
+        },
+        servers: [
+          {
+            url: if Apipie.configuration.swagger_api_host
+                   Apipie.configuration.swagger_api_host + Apipie.api_base_url(version)
+                 else
+                   Apipie.api_base_url(version)
+                 end
+          }
+        ],
+        paths: {},
+        components: {
+          schemas: {}
+        },
+        tags: []
```

Comment:
Some changes to the structure of the base schema object here. See https://swagger.io/specification/